app/notifications/publishers/job_application_data_expiry_notification.rb

```diff
@@ -1,6 +1,7 @@
 class Publishers::JobApplicationDataExpiryNotification < Noticed::Base
   include ActionView::Helpers::UrlHelper
   include GovukLinkHelper
+  include GovukVisuallyHiddenHelper
```

Comment:
I'm not sure it's necessary to explicitly include this, [I _think_ it should include itself by default](https://github.com/x-govuk/govuk-components/blob/main/app/helpers/govuk_visually_hidden_helper.rb#L15).

Reply:
Ah sorry, of course 👌🏽